web/modules/custom/joinup_comment/joinup_comment.module

```diff
@@ -22,9 +22,7 @@ function joinup_comment_comment_view_alter(array &$build, EntityInterface $entit
 
 /**
  * Implements hook_form_FORM_ID_alter().
- *
- * Change the submit button label for the comment form.
```

Comment:
Why is this documentation removed? It is not great but still better than having to read the code.